Italian/guardare/guarda.md

```diff
@@ -1,2 +1,12 @@
 Le pareti si restringono e dei simpatici e piccoli elfi ballano e cantano...
-i funghi "speciali" che ho mangiato ieri in realtà erano allucinogeni
```

Comment:
Done, I wrote the new sentence as a new line instead of at the end of the existing one.